content/en/dashboards/list/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Dashboard List
+kind: Documentation
+description: "Organize and manage dashboards with lists"
+disable_toc: false
+further_reading:
+- link: "dashboards/"
+  tag: "Documentation"
+  text: "Dashboards Overview"
+- link: "dashboards/guide/maintain-relevant-dashboards"
+  tag: "Guide"
+  text: "Best practices for maintaining relevant dashboards"
+---
+
+## Overview
+
+Take control of your expanding dashboard collection with Dashboard List features to organize and streamline effectively. Group dashboards into lists, assign them to specific teams, and mark important ones as favorites for fast access to key visualizations. Manage dashboard organization further by using functionalities like filtering by Teams, performing bulk actions for efficient management, and assigning Teams to multiple dashboards. Explore, create, and manage custom or integrated dashboards effortlessly on the [Dashboard List page][1].
+View and manage your dashboards:
+- [Use the *All Dashboards* table to sort, search, and group your lists.](#view-all-dashboards)
+- [Organize your dashboard views through lists.](#lists)
+
+## View all dashboards
+
+The **All Dashboards** table lists dashboards in your Datadog organization, either custom created or available as an out-of-the-box dashboard. Select multiple dashboards in the table to conduct bulk actions, such as setting [Teams](#teams) to dashboards or adding dashboards to [lists](#lists).
```

Comment:
```suggestion
The **All Dashboards** table lists dashboards in your Datadog organization, either custom created or available as an out-of-the-box dashboard. Select multiple dashboards in the table to conduct bulk actions, such as associating [Teams](#teams) with dashboards or adding dashboards to [lists](#lists).
```
Or assigning Teams to dashboards? I wasn't sure what "setting Teams to dashboards" meant.

Reply:
I've seen both in the docs, but your suggestions reads better to me.